docs/_sass/_docs-display-component.scss

```diff
@@ -14,6 +14,7 @@
     transition: background-color .25s;
 
     > .fd-tile__content {
+        width: 100%;
```

Comment:
I think this will only fix the problem on the docs site, no? Shouldn't we fix it in the actual scss?

Reply:
Fixed!